website/docs/tutorials/import-your-existing-apis/typegraph_std.py

```diff
@@ -0,0 +1,21 @@
+from typegraph import TypeGraph, policies
+from typegraph_std.github.ghes import import_ghes
+from typegraph_std.google.gmail import import_gmail
```

Comment:
I suggest to enable simpler imports: `from typegraph_std.google import import_gmail`